### ISSUE TITLE
Format adds/removes space after description on every run

### ISSUE THREAD

**Issue by jamesrenaud** [likec4/likec4#1074]
The doc format is behaving oddly with the `description` property within an element.

With this example:

```
someone = person 'Person' {
    description: 'A person'
  }
```

When formatting *once* the `.c4` file, a space is added to `description`

```
someone = person 'Person' {
    description : 'A person'
  }
```

Formatting a *second* time, the space is removed...

```
someone = person 'Person' {
    description: 'A person'
  }
```

If you have a mixed bag, it reverses itself

Pre-format
```
someone = person 'Person' {
    description : 'A person'
  }
someoneElse = person 'Person2' {
    description: 'Another person'
  }
```

Post-format
```
someone = person 'Person' {
    description: 'A person'

  }
someoneElse = person 'Person2' {
    description : 'Another person'
  }
```

Here's a live example: 

https://github.com/user-attachments/assets/ad485442-1bd9-4079-892d-fcc32cf1ddb5

**Comment by davydkov** [comment]
@pavelpykhtin I recall there was discussion on langium, formatting issues on second run.

**Comment by pavelpykhtin** [comment]
@davydkov , yes, I've checked that issue. It is already closed with fix.

This particular issue is related to two overlapping edits which the formatter is trying to apply: add space after a property name and remove space before a colon. If there is one or more space between a property name and a colon, then 'remove space' edit is applied and overwrites any other edits. But if there is no space, then the second edit is ignored and 'add one space' edit is applied.
I hoped that formatter solves such issues automatically, but it seems it's not the case. 

Thus, I added a check to ensure that there are no overlapping edits.
